serverless.yml

```diff
@@ -8,32 +8,33 @@ plugins:
 
 custom:
   domain:
-    dev: graphql-dev.gameifai.org
-    qa: graphql-qa.gameifai.org
-    prod: graphql.gameifai.org
+    dev: api-dev.gameifai.org
+    qa: api-qa.gameifai.org
+    prod: api.gameifai.org
   customDomain:
     domainName: ${self:custom.domain.${opt:stage, 'dev'}}
-    # basePath: 'graphql' # api root url becomes: api.mentorpal.org/graphql
+    basePath: 'graphql' # api root url becomes: api.mentorpal.org/graphql
```

Comment:
this will also need to be taken into consideration for updating target domains in gai-jupyter